.github/main.yml

```diff
@@ -0,0 +1,147 @@
+            #
+            # While this does modify the manifest file in-place, the changes are not
+            # commited to the repository, and only exist in the action's filesystem.
+            - name: Modify Module Manifest With Release-Specific Values
+              id: sub_manifest_link_version
+              uses: cschleiden/replace-tokens@v1
+              with:
+                  files: "system.json"
+              env:
+                  VERSION: ${{steps.get_version.outputs.version-without-v}}
+                  URL: ${{ env.project_url }}
+                  MANIFEST: ${{ env.latest_manifest_url }}
+                  DOWNLOAD: ${{ env.release_module_url }}
+
+            # Create a "module.zip" archive containing all the module's required files.
+            # If you have other directories or files that will need to be added to
+            # your packaged module, add them here.
+            - name: Create Module Archive
+              run: |
+                  # Note that `zip` will only emit warnings when a file or directory
+                  # doesn't exist, it will not fail.
+                  zip                                    \
+                    `# Options`                          \
+                    --recurse-paths                      \
+                    `# The name of the output file`      \
+                     ./system.zip                        \
+                    `# The files that will be included.` \
+                    fonts/                               \
+                    images/                              \
+                    lang/                                \
+                    lib/                                 \
+                    modules/                             \
+                    styles/                              \
+                    templates/                           \
+                    CHANGELOG.md                         \
+                    CONTRIBUTING.md                      \
+                    LICENSE.txt                          \
+                    README.md                            \
+                    package.json                         \
+                    system.json                          \
+                    template.json
+                  # Don't forget to add a backslash at the end of the line for any
+                  # additional files or directories!
+
+            # Update the GitHub release with the manifest and module archive files.
+            - name: Update Release With Files
+              id: create_version_release
+              uses: ncipollo/release-action@v1
+              with:
+                  allowUpdates: true
+                  name: ${{ github.event.release.name }}
+                  draft: ${{ github.event.release.unpublished }}
+                  prerelease: ${{ github.event.release.prerelease }}
+                  token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need to generate this secret or does it exist in the GitHub repo by default?